projects/sd-jwt-dotnet.md

```diff
@@ -0,0 +1,68 @@
+# Project Name
+
+SD-JWT-DotNet
+
+# Preferred Maturity Level
+
+Labs
+
+# Project Description
+
+A Dot Net implementation of the [Selective Disclosure for JWTs](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-04.html) (SD-JWT) specification.
+
+# Alignment with the OpenWallet Foundation Mission
+
+This library implements the Selective Disclosure for JWTs specification, which is an important building block for future digital identity wallets.
+
+# Code of Conduct
+
+None
+
+# TAC Sponsor
+
+None
+
+# Project License
+
+Apache 2.0.
+
+# Source Control
+
+[https://github.com/thomas-tran/sd-jwt-dotnet](https://github.com/thomas-tran/sd-jwt-dotnet)
+
+# Issue Tracker
+
+[https://github.com/thomas-tran/sd-jwt-dotnet/issues](https://github.com/thomas-tran/sd-jwt-dotnet/issues)
+
+# External Dependencies
+
+```nuget
+// https://www.nuget.org/packages/gfoidl.Base64
+// License: MIT
+gfoidl.Base64
+
+```
+
+# Release Methodology
+
+N/A
+
+# Initial Maintainers
+
+Thomas Tran([Github](https://github.com/thomas-tran))
+
+# Proposed Project Governance
+
+None
+
+# Links to Documented Governance Practices
+
+https://docs.google.com/forms/u/2/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/formResponse?pli=1
```

Comment:
Unfortunately, it looks like this points to a blank form for me. When you hear back from the Linux Foundation with a document that you agree with, you can update that here.

Reply:
I haven't got any email form the Linux Foundation yet, I will update it once I got it